src/mqtt.cpp

```diff
@@ -1,4 +1,5 @@
 #include "mqtt.h"
+#include "modbus.h"
```

Comment:
die mqtt klasse muss für sich alleine stehen und darf ausser zur config Klasse keine weiteren abhängigkeiten haben. Die mqtt Klasse wird noch in anderen Projekten benötigt. Die EInbindung der modbus klasse ist auch nicht notwendig, zirkelbeziehungen sollten icht entstehen.

Reply:
Ja, der include sollte nicht da sein - hatte ich wohl vorübergehend gebraucht und dann vergessen.